### PR TITLE
Tests are now parallel

### LINKED ISSUE #117: Tests are dependent on the same cache
Some tests fail because they operate on the same redis cache

### PR DESCRIPTION
**Pull request by MJHC**
closes #117

### CODE REVIEW

File: server/src/managers/websocket/connection.rs [hunk 1]
```diff
@@ -713,7 +713,7 @@ pub(crate) mod test {
                 .len(),
             1
         );
-
+        println!("here");
```
Comment (Darkros1245) [inline server/src/managers/websocket/connection.rs:716]: Remove

**Review by Darkros1245 — CHANGES_REQUESTED** [review]

File: server/src/managers/websocket/connection.rs (hunk 1 above)
Comment (alexsteffensen) [reply server/src/managers/websocket/connection.rs:716]: agree

File: server/src/test_utils/message_cache.rs [hunk 2]
```diff
@@ -9,8 +9,32 @@ pub fn generate_uuid() -> String {
     guid.to_string()
 }
 
-pub async fn teardown(mut con: deadpool_redis::Connection) {
-    cmd("FLUSHALL").query_async::<()>(&mut con).await.unwrap();
+pub async fn teardown(key: &str, mut con: deadpool_redis::Connection) {
+    let pattern = format!("{}presence::*", key);
```
Comment (alexsteffensen) [inline server/src/test_utils/message_cache.rs:13]:
```suggestion
    let pattern = format!("{}*", key);
```

**Review by alexsteffensen — APPROVED** [review]
Det ser meget godt ud

**Review by Darkros1245 — APPROVED** [review]

**Pull request merged**